Editor/Scripts/Startup/RegressionPackagePopup.cs

```diff
@@ -0,0 +1,265 @@
+    private static RegressionPackagePopup window;
+    private static bool loggedIn = false;
+    private static string email = "";
+    private static string password = "";
+    
+    void OnEnable()
+    {
+        string packagePath = "Packages/gg.regression.unity.bots/Editor/Images/banner.png";
+        string assetsPath = "Assets/Editor/Images/banner.png";
+
+        bannerImage = AssetDatabase.LoadAssetAtPath<Texture2D>(packagePath);
+    
+        if (bannerImage == null)
+        {
+            bannerImage = AssetDatabase.LoadAssetAtPath<Texture2D>(assetsPath);
+        }
+
+        if (bannerImage == null)
+        {
+            Debug.LogWarning("Failed to load banner image");
+        }
+    }
+    
+    [MenuItem("Regression Games/Setup")]
+    public static async void ShowWindow()
+    {
+        // attempt a login with saved credentials
+        email = RGSettings.GetOrCreateSettings().GetEmail();
+        password = RGSettings.GetOrCreateSettings().GetPassword();
+        await Login();
+        
+        if (window == null)
+        {
+            Rect windowRect = new Rect(100, 100, 600, 600);
+            window = (RegressionPackagePopup)EditorWindow.GetWindowWithRect(typeof(RegressionPackagePopup), windowRect, true, "Welcome to Regression Games!");
+        }
+        else
+        {
+            window.Focus();
+        }
+    }
+
+    void OnGUI()
+    {
+        if (!loggedIn)
+        {
+            RenderLoginScreen();
+        }
+        else
+        {
+            RenderWelcomeScreen();
+        }
+    }
+
+    private void RenderLoginScreen()
+    {
+        if (bannerImage != null)
+        { 
+            GUILayout.Box(bannerImage, GUILayout.ExpandWidth(true), GUILayout.Height(200));
+        }
+        
+        // H1
+        GUIStyle h1Style = new GUIStyle(EditorStyles.largeLabel);
+        h1Style.fontSize = 20;
+        h1Style.fontStyle = FontStyle.Bold;
+        h1Style.normal.textColor = Color.white;
+        
+        // P
+        GUIStyle pStyle = new GUIStyle(EditorStyles.label);
+        pStyle.fontSize = 12;
+        pStyle.wordWrap = true;
+
+        // Title
+        GUILayout.BeginHorizontal();
+        GUILayout.FlexibleSpace();
+        GUILayout.Label("Welcome to Regression Games!", h1Style, GUILayout.Width(550));
+        GUILayout.FlexibleSpace();
+        GUILayout.EndHorizontal();
+        GUILayout.Space(10);
+
+        // Description
+        GUILayout.BeginHorizontal();
+        GUILayout.FlexibleSpace();
+        GUILayout.Label(
+            "Regression Games Unity Bots is an SDK that makes it easy to integrate powerful and useful bots into " +
+            "your game. Build bots to QA test your game, act as NPCs, compete against players in multiplayer, and " +
+            "determine optimal game balance. Our integration patterns, ready-to-go bots (coming soon), and samples " +
+            "make it easy to get started.", pStyle, GUILayout.Width(550));
```

Comment:
We've said "make it easy" already, so  @switch up the wording a bit
```suggestion
            "will help you get started quickly.", pStyle, GUILayout.Width(550));
```

Reply:
Fixed